### PR TITLE
Create miniCore.md

### PR DESCRIPTION
**Pull request by rat10**
This captures my interpretation of the current status of the WG efforts and extends it to a proposal on how to proceed.

### CODE REVIEW

File: docs/miniCore.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+### Extensions
+
+#### RDF-star
+
+Of course getting rid of the RDF standard reification quad was *the* motivation for the RDF-star effort. 
+However, it requires a considerable extension to the core of RDF and is still struggling in a lot
+of crucial details. I propose therefore to make it a semantic extension to RDF - endorsed by this WG,
+but still separate and optional. A store deciding to support RDF-star would be welcome to map the 
+annotation syntax to triple terms, not to standard reification quads.
```
Comment (pchampin) [inline docs/miniCore.md:170]: There are two things that do not quite make sense in this sentence, with respect to how semantic extensions are defined in RDF Semantics [1]:

* semantic extensions do not augment the abstract syntax, so if triple terms are not part of the abstract syntax, there is no way for a semantic extension to map anything to triple terms;

* the semantics of RDF is defined purely on the abstract syntax, it has no access to the concrete syntax. Since annotation are only syntactic sugar in the concrete syntax and have no counterpart in the abstract syntax, semantic extensions can not say anything about annotations.

What could be done, however, would be to define a semantic extension where ill-formed instances of rdf:Statements (e.g. statements with 0 or 2 subjects) are forbidden by the *syntactic restrictions* of this extension. Statements would still be regular IRIs or bnodes, not triple terms, but at least they would be guaranteed to always have a subject, a predictae and an object (and only one of each).

[1] https://www.w3.org/TR/rdf11-semantics/#dfn-semantic-extension

File: docs/miniCore.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
+# A Minimal Core and Clearly Defined Extension Points
```
Comment (afs) [inline docs/miniCore.md:1]: Please add author name and date.

Then "I" makes sense!

**Review by TallTed — CHANGES_REQUESTED** [review]
This is incomplete, but I think still an improvement.

File: docs/miniCore.md (hunk 1 above)
Comment (rat10) [reply docs/miniCore.md:170]: I see, so introducing a new term into the model is not a semantic extension anymore. IIUC, a  syntax that supports syntactic sugar for reification could be accompanied by a semantic extension as you outline it. I don't follow why some insist that the model needs to be extended with a new term type. IMO such a syntax, and syntactic sugar like triple terms for N-Triples, and clearly defined restrictions on the mapping to RDF standard refication, would provide enough common ground and would leave implementations free to choose if they implement it via RDF standard reification or RDF-star or named graphs or whatever.

File: docs/miniCore.md (hunk 2 above)
Comment (rat10) [reply docs/miniCore.md:1]: It's still written as an email I fear. However, if only I knew how to modify my own pull request...

File: docs/miniCore.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+This proposal aims to define a minimal core and sensible extension points.
+
+
+## Consensus on Annotation Syntax
```
[the remark this answers is not in the document]
Comment (rat10) [reply docs/miniCore.md:6]: Different, but not necessarily better. I'm reluctant to go into such detailed editing as long as the broad lines are not discussed.

File: docs/miniCore.md (hunk 2 above)
Comment (afs) [reply docs/miniCore.md:1]: Edit your copy and add commits to the branch [rat10:patch-6](https://github.com/rat10/rdf-star-wg/tree/patch-6).

Then either rebase locally to clean up, or we can squash to one commit at the point of merge.

File: docs/miniCore.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+
+December 2023 we found a possible position around the annotation syntax (sometimes also called shorthand syntax) 
+with added naming to support multi-edges:
+
+```turtle
+:s :p :o {| :u :v |}          # an `spo` edge, implicitly named by a blank node
```
[the remark this answers is not in the document]
Comment (rat10) [reply docs/miniCore.md:12]: Too much white space inside the annotation, and I'm not a fan of adding detail like prefix declarations when the example itself is very clearly just an example and should concentrate on what is new or different.

File: docs/miniCore.md (hunk 2 above)
Comment (rat10) [reply docs/miniCore.md:1]: Thanks. I just saw that TallTed already provided a patch.

File: docs/miniCore.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (TallTed) [reply docs/miniCore.md:8]: "Consensus" suggests that everyone has already agreed on this approach. I think that's a significantly premature label, and could easily cause the remainder of the document to be rejected on principle. I don't think that's what you want, so I suggest again that you change this heading.

File: docs/miniCore.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (TallTed) [reply docs/miniCore.md:14]: Please either include the prefix declaration, or do something like `ex:s` instead of `:s`. It is *VERY* easy for people to read examples as fully-baked content, which these are not, and that leads to all sorts of misunderstandings.

As to the whitespace, it's intended to make the columns align visually, so it's easy to see the differences and progressions between lines. *You* know what's intended, so it seems like extraneous whitespace. *I* and others are trying to understand what you intended, and this syntactically optional but hopefully permitted whitespace helps with that.

[the remark this answers is not in the document]
Comment (rat10) [reply docs/miniCore.md:14]: I consider all three suggestions - adding a prefix declaration, adding 'ex:' to every term and adding extra whitespace - as superfluous and distracting from the content of the example. However is lured by such terseness into thinking this was actual working RDF lacks the most basic understanding of Turtle and is not the intended audience of this post. We can't speak to everybody all the time.

File: docs/miniCore.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (rat10) [reply docs/miniCore.md:8]: I wasn't aware that people are so picky about wording in a discussion document. The idea was to make discussions easier to follow by moving them from the mailinglist into the Github wiki. If we now start to treat them like official WG documents and discuss finegrained editorial changes, the discussion becomes equally hard to follow, and unproductive w.r.t. to the discussions we actually need to have. 
That said: would you rephrase your suggested resolution into "Annotation Syntax as a Common Ground"?

[the remark this answers is not in the document]
Comment (TallTed) [reply docs/miniCore.md:8]: People *in* the WG will interpret things differently than people *outside* the WG, and this document is visible to *all*. For this simple reason, we should be careful not to write things (like titles including "consensus" without "possible" or other qualifier) that imply support/acceptance by the entire WG until such time as that implication is accurate. Your suggested rephrasing is similarly problematic — it presents a *possibility* as if it had already been decided. Perhaps "A New Annotation Syntax as a Possible Common Ground" would satisfy you (as it would me)?

File: docs/miniCore.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (TallTed) [reply docs/miniCore.md:14]: Sloppiness begets sloppiness. Failing to include all necessary pieces of a syntax in development discussion often leads to misunderstanding and/or forgetting about some impacts on or impacts of those omitted pieces of syntax. This can lead to entirely broken "final" documents that need radical revision when the time comes to request a status change to Candidate Recommendation or Proposed Recommendation. I have been in groups where this has happened. It is my hope to never again experience anything similar.

File: docs/miniCore.md [hunk 5]
```diff
@@ -0,0 +1,194 @@
+# A Minimal Core and Clearly Defined Extension Points
+
+This proposal aims to define a minimal core and sensible extension points.
+
+
+## Consensus on Annotation Syntax
+
+December 2023 we found consensus around the annotation syntax (sometimes also called shorthand syntax)
```
[the remark this answers is not in the document]
Comment (TallTed) [reply docs/miniCore.md]: > This captures my interpretation of the current status of the WG efforts and extends it to a proposal on how to proceed.

The moment you "[extended] it to a proposal on how to proceed" is the moment all your personal commentary should have been rephrased. I suggested such rephrasing in the first part of the document, but you rejected my suggestions along these lines.

I think you need to decide whether this document will reflect a personal perspective, or represent a proposed WG path forward; and then adjust it throughout to match your decision.

File: docs/miniCore.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (rat10) [reply docs/miniCore.md:8]: The annotation syntax is not new.

File: docs/miniCore.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (rat10) [reply docs/miniCore.md:14]: I will not engage in this discussion any more and I would ask you to respect my decision and concentrate on discussing what I wrote, not on the style in which I wrote it.

File: docs/miniCore.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (TallTed) [reply docs/miniCore.md:8]: Changed suggestion to "The Annotation Syntax as a Possible Common Ground".
